src/roulette.py

```diff
@@ -49,10 +51,78 @@ def winAmount(self, amount: float) -> float:
 
 class Bin(frozenset):
     """
-    :class:`Bin` contains a collection of :class:`Outcome` instances which reflect the winning bets that are paid for a
-    particular bin on a Roulette wheel. In Roulette, each spin of the wheel has a number of :class:`Outcome` instances.
-    Example: A spin of 1, selects the “1” bin with the following winning :class:`Outcome` instances: “1” , “Red”,
-    “Odd”,“Low” , “Column 1” , “Dozen 1-12” , “Split 1-2” , “Split 1-4” , “Street 1-2-3” , “Corner 1-2-4-5”, “Five Bet”,
-    “Line 1-2-3-4-5-6” , “00-0-1-2-3” , “Dozen 1”, “Low” and “Column 1”. These are collected into a single :class:`Bin`.
+    :class:`Bin` contains a collection of :class:`Outcome` instances which reflect the winning bets
+    that are paid for a particular bin on a Roulette wheel. In Roulette, each spin of the wheel
+    has a number of :class:`Outcome` instances. Example: A spin of 1, selects the “1” bin with
+    the following winning :class:`Outcome` instances: “1” , “Red”, “Odd”,“Low” , “Column 1” ,
+    “Dozen 1-12” , “Split 1-2” , “Split 1-4” , “Street 1-2-3” , “Corner 1-2-4-5”, “Five Bet”,
+    “Line 1-2-3-4-5-6” , “00-0-1-2-3” , “Dozen 1”, “Low” and “Column 1”. These are collected into a
+    single :class:`Bin`.
+    """
+
+
+class Wheel:
+    """
+    :class:`Wheel` contains the 38 individual bins on a Roulette wheel, plus a random number
+    generator. It can select a :class:`Bin` at random, simulating a spin of the Roulette wheel.
+
+    .. attribute:: bins
+
+        Contains the individual Bin instances.
+
+        This is a tuple of 38 elements. This can be built with ``tuple(Bin() for i in
+        range(38))``
+
+    .. attribute:: rng
+
+        A random number generator to select a :class:`Bin` from the **bins** collection.
+
+        For testing, we’ll often want to seed this generator. For simulation processing, we can
+        set the seed value using ``os.urandom()``.
     """
-    pass
+
+    def __init__(self) -> None:
+        """
+        Creates a new wheel with 38 empty Bin instances. It will also create a new random number
+        generator instance.
+
+        At the present time, this does not do the full initialization of the Bin instances. We’ll
+        rework this in a future exercise.
+        """
+        self.bins = tuple(Bin() for i in range(38))
```

Comment:
try to avoid magic literals. don't need to change for this one as the doc string is explaining what magic literal 38 is about.

Reply:
Noted. Thanks.